nemo/collections/asr/parts/k2/maploss.py

```diff
@@ -0,0 +1,269 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
filename -> map_loss.py

Reply:
Done in [f0308a1](https://github.com/NVIDIA/NeMo/pull/3351/commits/f0308a1a8e24f93b1ed3e00501a7a5ef1a6f4e55)